GetIntoTeachingApi/Models/TeacherTrainingAdviserSignUp.cs

```diff
@@ -35,7 +38,18 @@ public class TeacherTrainingAdviserSignUp
         public DateTime? DateOfBirth { get; set; }
         public string TeacherId { get; set; }
         public string DegreeSubject { get; set; }
-        public string AddressTelephone { get; set; }
+        public string AddressTelephone
+        {
```

Comment:
Is it worth using the arrow syntax to save some lines?

```c#
 get => return SanitizeTelephone(_addressTelephone);
 set => _addressTelephone = value;
```

Reply:
Yeah that's a good shout, I'll maybe see if I can get the linter setup to catch that